INTRODUCTION.md

```diff
@@ -1591,10 +1588,9 @@ so the protocol will be enabled by default only with a librdkafka major release.
 
  - Protocol rebalance is fully incremental, so the only allowed functions to
    use in a rebalance callback will be `rd_kafka_incremental_assign` and
-   `rd_kafka_incremental_unassign`. Currently you can still use existing code
-   and the expected function to call is determined based on the chosen
-   `partition.assignment.strategy` but this will be removed in next
-   release.
+   `rd_kafka_incremental_unassign`. In the `classic` protocol, the expected 
+   function to call is determined based on the chosen `partition.assignment.strategy`
+   but this is removed for the `consumer` protocol.
 
```

Comment:
Below when it says "Currently the timeout" ... change it to:

>    Currently the timeout is a fixed one and corresponds to the default remote session
   timeout in Apache Kafka but it'll be possible to pass a custom timeout
   when calling `close` (KIP-1092).

Reply:
>  - A `TOPIC_AUTHORIZATION_FAILED` error will be returned as was happening
   with group protocol `classic`, it happens once per heartbeat or subscription change and
   it's a single error for the whole subscription. It's reported even when only a single topic
   isn't authorized.

Reply:
In `Supported protocol versions` set the max supported version to 1